pkg/store/container/container.go

```diff
@@ -142,5 +157,10 @@ func (s *Store) List() []Container {
 func (s *Store) Delete(id string) {
 	s.lock.Lock()
 	defer s.lock.Unlock()
+	id, err := s.idIndex.Get(id)
+	if err != nil {
+		return
+	}
+	_ = s.idIndex.Delete(id)
```

Comment:
I feel like it's fine to call `s.idIndex.Delete` directly and ignore the error if we don't care about it anyway.
You may want to add `nolint: errcheck`.

Reply:
done